app/views/rounds/_pairings.html.slim

```diff
@@ -72,7 +72,7 @@
               div class='form-group'
                 div class='form-check form-check-inline'
                   = f.check_box :intentional_draw, class: 'form-check-input'
-                  = f.label :intentional_draw, 'Intentional Draw', class: 'form-check-label'
+                  = f.label :intentional_draw, 'ScIntentional Draw', class: 'form-check-label'
```

Comment:
It looks like this was committed accidentally?

Reply:
ha!  My debugging is showing.  :)